packages/create-sitecore-jss/src/templates/nextjs/src/middleware.ts

```diff
@@ -7,6 +7,14 @@ export default async function (req: NextRequest, ev: NextFetchEvent) {
 }
 
 export const config = {
-  // Exclude Sitecore editing API routes
-  matcher: ['/', '/((?!api/editing/).*)'],
+  /*
+  * Match all paths except for:
+  * 1. /api routes
+  * 2. /_next (Next.js internals)
+  * 3. /sitecore/api (Sitecore API routes)
+  * 4. /- (Sitecore media)
+  * 5. /layouts/system (Sitecore Tracking API)
```

Comment:
I would suggest we leave this one out since it only applies to SXP tracking?

Reply:
Make sense for me